packages/cli/package.json

```diff
@@ -25,11 +25,13 @@
     "access": "public"
   },
   "dependencies": {
-    "@babel/core": "^7.6.0",
-    "@babel/plugin-transform-runtime": "^7.6.2",
-    "@babel/preset-env": "^7.6.0",
-    "@babel/runtime": "^7.7.2",
+    "@babel/core": "^7.8.3",
+    "@babel/plugin-syntax-dynamic-import": "^7.8.3",
+    "@babel/plugin-transform-runtime": "^7.8.3",
+    "@babel/preset-env": "^7.8.3",
+    "@babel/runtime": "^7.8.3",
     "@webcomponents/webcomponentsjs": "^2.3.0",
+    "babel-eslint": "^10.0.3",
```

Comment:
Oh, my bad, this shouldn't be a dependency (dev or otherwise) of the CLI.  ESLint is a dev tool, but also not used by the CLI, so should be moved to the root instead.  (where **eslint** currently lives)

Reply:
Also, I was able to remove this and not have any issues
```
    "padding-line-between-statements": [
      "error",
      { "blankLine": "always", "prev": ["const", "let", "var"], "next": "*"},
      { "blankLine": "any",    "prev": ["const", "let", "var"], "next": ["const", "let", "var"]}
    ],
```

Also, based on **babel-eslint**, it is recommended to pass a _babel.config.js_.